xapian-letor/tests/api_letor.cc

```diff
@@ -107,6 +107,8 @@ DEFINE_TESTCASE(preparetrainingfile, generated)
     string qrel = data_directory + "qrel.txt";
     Xapian::prepare_training_file(db_path, query, qrel, 10,
 				  "training_output.txt");
+
+    return true;
```

Comment:
Would it be a good idea to do more than test that `::prepare_training_file()` doesn't raise an exception or abort? We should test that it produces the desires output.